lfs/pointer_clean.go

```diff
@@ -31,7 +31,8 @@ func PointerClean(reader io.Reader, fileName string, fileSize int64, cb CopyCall
 	var oid string
 	var size int64
 	var tmp *os.File
-	var exts []*PointerExtension
+	// Initialise to empty set rather than nil for consistency with pointer decode
```

Comment:
Which assert would fail in this case? If I change the line back I don't get any failures here. It shouldn't matter if a slice is really `nil`, it can always be treated as a slice with empty checks being done with `len()`. I don't think anything should be comparing a slice with `nil` directly.

Aside from that, I don't think the comment adds much clarity. I also think we should prefer `:=` to `var =` when initializing with a value.